G/gmime/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gmime"
+version = v"3.2.15"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/jstedfast/gmime/releases/download/$(version)/gmime-$(version).tar.xz", "84cd2a481a27970ec39b5c95f72db026722904a2ccf3fdbd57b280cf2d02b5c4")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gmime*
+echo '#define ICONV_ISO_INT_FORMAT "iso-%u-%u"' > iconv-detect.h
+echo '#define ICONV_ISO_STR_FORMAT "iso-%u-%s"' >> iconv-detect.h
+echo '#define ICONV_SHIFT_JIS "shift-jis"' >> iconv-detect.h
+echo '#define ICONV_10646 "UCS-4BE"' >> iconv-detect.h
+
+export LDFLAGS="-liconv"
```

Comment:
The fact that you need to do this manually frankly looks to me an upstream bug.

Reply:
This is a message after `./configure --prefix=${prefix}   --build=${MACHTYPE}   --host=${target}`
```
checking preferred charset formats for system iconv... cannot run test program while cross compiling
configure: error: Copy target system's iconv-detect.h to . and configure with ac_cv_have_iconv_detect_h=yes
```
So what added iconv-detect.h manually

Reply:
And in fact I can compile without and explicit `LDFLAGS="-liconv"`